src/main/scala/tech/cryptonomic/conseil/routes/Service.scala

```diff
@@ -0,0 +1,42 @@
+package tech.cryptonomic.conseil.routes
+
+import akka.http.scaladsl.marshalling.{PredefinedToEntityMarshallers, ToEntityMarshaller}
+import akka.http.scaladsl.model.MediaTypes
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.{Route, StandardRoute}
+import com.typesafe.config.Config
+import com.typesafe.scalalogging.LazyLogging
+import tech.cryptonomic.conseil.tezos.ServiceOperations
+import tech.cryptonomic.conseil.util.JsonUtil._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object Service {
+  def apply(config: Config)(implicit apiExecutionContext: ExecutionContext): Service = new Service(config)
+}
+
+class Service(config: Config)(implicit apiExecutionContext: ExecutionContext) extends LazyLogging {
```

Comment:
I wanted another class because I assumed its different functionality, and I didn't want to mix them there. Regarding handling other routes in different classes I think it's good idea, but not necessary now, as those routes are not that big. If they grow we should definitely think about it.